bot/app.py

```diff
@@ -20,7 +20,7 @@ async def shutdown(self) -> None:
 
 
 def main() -> None:
-    app = hikari.GatewayBot(CONFIG.TOKEN, intents=INTENTS)
+    app = hikari.GatewayBot(CONFIG.TOKEN, intents=INTENTS, banner="crescent")
```

Comment:
This is out of scope for this PR. Personally I prefer to keep the hikari banner, but I'm willing to discuss is in an issue and possibly change it in a separate pr

```suggestion
    app = hikari.GatewayBot(CONFIG.TOKEN, intents=INTENTS)
```

Reply:
ohhh fuck i didn't think i saved that lmao